.test-infra/jenkins/job_LoadTests_GBK_Flink_Go.groovy

```diff
@@ -0,0 +1,223 @@
+        iterations          : 1,
+        fanout              : 8,
+        parallelism         : 16,
+        input_options        : """
+                                   {
+                                     "num_records": 5000000,
+                                     "key_size": 10,
+                                     "value_size": 90
+                                   }
+                               """.trim().replaceAll("\\s", ""),
+        endpoint             : 'localhost:8099',
+        environment_type     : 'DOCKER',
+        environment_config   : "${DOCKER_CONTAINER_REGISTRY}/beam_go_sdk:latest",
+      ]
+    ],
+    [
+      title          : 'Load test: reiterate 4 times 10kB values',
+      test           : 'group_by_key',
+      runner         : CommonTestProperties.Runner.FLINK,
+      pipelineOptions: [
+        job_name             : "load-tests-go-flink-batch-gbk-1-${now}",
+        influx_measurement   : 'go_batch_gbk_1',
+        iterations          : 4,
+        fanout              : 1,
+        parallelism         : 5,
+        input_options        : """
+                                   {
+                                     "num_records": 5000000,
+                                     "key_size": 10,
+                                     "value_size": 90,
+                                     "num_hot_keys": 200,
+                                     "hot_key_fraction": 1
+                                   }
+                               """.trim().replaceAll("\\s", ""),
+        endpoint             : 'localhost:8099',
+        environment_type     : 'DOCKER',
+        environment_config   : "${DOCKER_CONTAINER_REGISTRY}/beam_go_sdk:latest",
+      ]
+    ],
+    [
+      title          : 'Load test: reiterate 4 times 2MB values',
+      test           : 'group_by_key',
+      runner         : CommonTestProperties.Runner.FLINK,
+      pipelineOptions: [
+        job_name             : "load-tests-go-flink-batch-gbk-1-${now}",
+        influx_measurement   : 'go_batch_gbk_1',
+        iterations          : 4,
+        fanout              : 1,
+        parallelism         : 5,
+        input_options        : """
+                                   {
+                                     "num_records": 20000000,
+                                     "key_size": 10,
+                                     "value_size": 90,
+                                     "num_hot_keys": 10,
+                                     "hot_key_fraction": 1
+                                   }
+                               """.trim().replaceAll("\\s", ""),
+        endpoint             : 'localhost:8099',
+        environment_type     : 'DOCKER',
+        environment_config   : "${DOCKER_CONTAINER_REGISTRY}/beam_go_sdk:latest",
+      ]
+    ],
+  ].each { test -> test.pipelineOptions.putAll(additionalPipelineArgs) }
+}
+
+def loadTestJob = { scope, triggeringContext, mode ->
+  def numberOfWorkers = 5
```

Comment:
Some of your tests need the parallelism of 16, but your Flink deployment is going to have only 5 task managers. You need to group your tests by parallelism, execute the first group, then rescale the cluster and execute the second group. Please take a look at `.test-infra/jenkins/job_LoadTests_Combine_Flink_Python.groovy` where such a pattern is being used.